src/knx/application_layer.h

```diff
@@ -39,12 +40,12 @@ class ApplicationLayer
      *        See 3.1.1 of @cite knx:3/3/7
      *        
      * @param apdu The submitted APDU.
-     * 
+     *
      * @param priority The ::Priority of the received request.
      * 
      * @param hopType Should routing be endless or should the NetworkLayer::hopCount be used? See also ::HopCountType.
      */
-    void dataGroupIndication(HopCountType hopType, Priority priority, uint16_t tsap, APDU& apdu);
+    virtual void dataGroupIndication(HopCountType hopType, Priority priority, uint16_t tsap, APDU& apdu) {dataGroupIndication(hopType, priority, tsap, apdu, noSecurity);}
```

Comment:
I try to keep the header files clean from code.  So these should go to the cpp-files later.

Reply:
Ok, will move it.

Reply:
done